### PR TITLE
Added ability to run all the benchmarks for a single model

### PR DESCRIPTION
**Pull request by piyushghai**
*Issue #, if available:*

*Description of changes:*
Improved scripts for Benchmark AI.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: benchmarks/lstm_ip.json [hunk 1]
```diff
@@ -0,0 +1 @@
+"[{\"input_sentence\": \"on the exchange floor as soon as ual stopped trading we <unk> for a panic said one top floor trader\"}]"
```
Comment (zachgk) [inline benchmarks/lstm_ip.json]: Missing newline

File: benchmarks/benchmark.py [hunk 2]
```diff
@@ -472,7 +474,12 @@ def modify_config_props_for_mms(pargs):
 
     modify_config_props_for_mms(pargs)
 
-    if pargs.all:
+    if pargs.model:
```
Comment (zachgk) [inline benchmarks/benchmark.py]: pargs.model has a default so this makes it impossible to reach the else case (run single benchmark on single model).  Add a new argument to the "target" mutually exclusive group for this additional case

File: benchmarks/benchmark.py [hunk 3]
```diff
@@ -472,7 +474,12 @@ def modify_config_props_for_mms(pargs):
...
+        benchmark_model = pargs.model[0].lower()
+        for benchmark_name in ['latency', 'throughput']:
```
Comment (zachgk) [inline benchmarks/benchmark.py]: Define the set of benchmarks as a constant for consistency

File: benchmarks/benchmark.py (hunk 2 above)
Comment (piyushghai) [reply benchmarks/benchmark.py]: Done.

File: benchmarks/benchmark.py (hunk 3 above)
Comment (piyushghai) [reply benchmarks/benchmark.py]: Done.

File: benchmarks/lstm_ip.json (hunk 1 above)
Comment (piyushghai) [reply benchmarks/lstm_ip.json]: As discussed offline, this is not needed here.

File: benchmarks/benchmark.py [hunk 4]
```diff
@@ -444,6 +448,7 @@ def modify_config_props_for_mms(pargs):
     target = parser.add_mutually_exclusive_group(required=True)
     target.add_argument('name', nargs='?', type=str, choices=benchmark_name_options, help='The name of the benchmark to run')
     target.add_argument('-a', '--all', action='store_true', help='Run all benchmarks')
+    target.add_argument('--model-name', nargs=1, type=str, default=[MODEL_RESNET_18], choices=MODEL_MAP.keys(), help='A preloaded model to run.  It defaults to {}. Use this argument to run all the benchmarks on this single model'.format(MODEL_RESNET_18))
```
Comment (zachgk) [inline benchmarks/benchmark.py]: Having this accept a model means there are two ways to pass in the model.  It is clearer to instead make this argument take a boolean similar to --all that simply changes the mode, and then reuse the --model/--custom-model parameter.  This better matches the behavior of name and all, as well as enables running the set of tests on custom models.  Maybe name running the set of tests for a given model "--suite"?

Comment (piyushghai) [reply benchmarks/benchmark.py]: Good point. 
Done.

**Pull request merged**